frontend/src/views/modelers/ModelersLoadedView.tsx

```diff
@@ -0,0 +1,203 @@
+  },
+}));
+export const ModelersLoadedView = ({ modelers, onModelerUpdated }) => {
+  const { projectId } = useParams();
+  const [{ context }, dispatch] = useMachine<ModelersLoadedViewContext, ModelersLoadedViewEvent>(
+    modelersLoadedViewMachine
+  );
+  const { modalDisplayed, menuAnchor, modeler } = context;
+  const classes = useStyles();
+
+  const onMore = (event, modeler) => {
+    let openMenu = { type: 'OPEN_MENU', menuAnchor: event.currentTarget, modeler } as OpenMenuEvent;
+    dispatch(openMenu);
+  };
+
+  const onRename = () => {
+    let openDialog = { type: 'OPEN_DIALOG', modalDisplayed: 'RenameModeler' } as OpenDialogEvent;
+    dispatch(openDialog);
+  };
+
+  const onPublish = () => {
+    let openDialog = { type: 'OPEN_DIALOG', modalDisplayed: 'PublishModeler' } as OpenDialogEvent;
+    dispatch(openDialog);
+  };
+
+  const onCloseModal = () => {
+    dispatch({ type: 'CLOSE_DIALOG' });
+    onModelerUpdated();
+  };
+  const onCloseContextMenu = () => dispatch({ type: 'CLOSE_MENU' });
+  let contextMenu = null;
+  if (menuAnchor) {
+    contextMenu = (
+      <ModelerContextMenu
+        menuAnchor={menuAnchor}
+        onCloseContextMenu={onCloseContextMenu}
+        onRename={onRename}
+        onPublish={onPublish}
+        modeler={modeler}
+      />
+    );
+  }
+  let modal = null;
+  if (modalDisplayed === 'RenameModeler') {
+    modal = (
+      <RenameModelerModal
+        modelerId={modeler.id}
+        initialModelerName={modeler.name}
+        onModelerRenamed={onCloseModal}
+        onClose={onCloseModal}
+      />
+    );
+  } else if (modalDisplayed === 'PublishModeler') {
+    modal = <PublishModelerModal modelerId={modeler.id} onModelerPublished={onCloseModal} onClose={onCloseModal} />;
+  }
+
+  const sortedModelers = [...modelers].sort((a, b) => {
+    if (a.name < b.name) {
+      return -1;
+    } else if (a.name > b.name) {
+      return 1;
+    } else {
+      return 0;
+    }
+  });
+
+  return (
+    <View>
+      <ModelersViewContainer>
+        <TableContainer className={classes.tableContainer}>
+          <Table data-testid="modelers">
+            <TableHead>
+              <TableRow>
+                <TableCell variant="head">Name</TableCell>
+                <TableCell variant="head">Status</TableCell>
+                <TableCell variant="head"></TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {sortedModelers.map((modeler) => (
+                <ModelerTableRow modeler={modeler} modelerURL={`/projects/${projectId}/edit`} onMore={onMore} />
+              ))}
+            </TableBody>
+          </Table>
+        </TableContainer>
+      </ModelersViewContainer>
+      {contextMenu}
+      {modal}
+    </View>
+  );
+};
+
+const ModelerTableRow = ({ modeler, modelerURL, onMore }) => {
+  return (
+    <TableRow key={modeler.id}>
+      <TableCell component="th" scope="row">
+        <Link href={modelerURL}>{modeler.name}</Link>
```

Comment:
This will trigger a refresh of the whole application, you should look at the example of ListItemLink I gave you instead.